bobtemplates/plone/content_type.py

```diff
@@ -143,6 +151,46 @@ def _update_rolemap_xml(configurator):
     update_file(configurator, file_path, match_str, insert_str)
 
 
+def _update_ct_types_fti_xml(configurator):
+    if (
+        (not configurator.variables.get('dexterity_type_global_allow', False)) and   # NOQA: E501
+        configurator.variables.get('dexterity_parent_container_type_name', False)    # NOQA: E501
+    ):
```

Comment:
i think the brackets for the if statement are not needed here.

Reply:
you mean between two `and` case? Save check 😅

Reply:
I mean:

`if not configurator.variables.get('dexterity_type_global_allow', False) and configurator.variables.get('dexterity_parent_container_type_name', False):`

and maybe better write it in a variable before:
parent_ct_type_name = configurator.variables.get('dexterity_parent_container_type_name', False) to make it shorter.

Reply:
also:
`configurator.variables.get('dexterity_type_global_allow') `
instead of:
`configurator.variables.get('dexterity_type_global_allow', False)`

get will return None in case there is not setting, which is fine.

Reply:
On it!